ui/selectors/selectors.test.js

```diff
@@ -2163,4 +2163,19 @@ describe('#getConnectedSitesList', () => {
       expect(getCurrentChainIdSpy).not.toHaveBeenCalled(); // Ensure overrideChainId is used
     });
   });
+
+  describe('#getRemoteFeatureFlags', () => {
+    it('returns null when featureFlagName does not exist in remoteFeatureFlags', () => {
```

Comment:
This test doesn't appear to reference anything called `featureFlagName`

Reply:
🕵🏻‍♂️  added 545943e3a603e160da198d220ddd14c21213b347